MvvmCross/Core/Binding/MvvmCross.Binding.csproj

```diff
@@ -32,6 +32,9 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
+    <Reference Include="System.Collections.Concurrent">
+      <HintPath>C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETPortable\v4.5\Profile\Profile7\System.Collections.Concurrent.dll</HintPath>
```

Comment:
I think it's safe to remove WP 8.0.  I believe ReactiveUI is also dumping it. See https://github.com/reactiveui/ReactiveUI/issues/945